src/sp/it/pl/audio/tagging/Metadata.kt

```diff
@@ -619,21 +620,18 @@ class Metadata: Item, Serializable {
         null
     }
 
-    private fun readArt(): Artwork? {
-        val af = if (isFileBased()) getFile().readAudioFile().orNull() else null
-        return af?.tag?.firstArtwork
-    }
+    private fun readArt(): Artwork? = getFile()?.let { it.readAudioFile().orNull() }?.tag?.firstArtwork
```

Comment:
should this really be named `read` and not `get`?

Reply:
getX nomenclature is an abomination and should never be used for things that compute, let alone blocking calls. readArt(), computeArt(),obtain, receive, retrieve, etc are all fine. get is not. In Kotlin get() even receives special treatment (can be invoked as []) - it definitely should be used carefully, because get() should be an accessor, not computer.

If you have better name, im open to suggestion, perhaps we can settle on a specific style. But getArt() is a no go.